presto-docs/src/main/sphinx/functions/noisy.rst

```diff
@@ -0,0 +1,200 @@
+=========================
+Noisy Aggregate Functions
+=========================
+
+Noisy aggregate functions are aggregate functions that provide random, noisy
+approximations of common aggregations like :func:`sum`, :func:`count`, and
+:func:`approx_distinct` as well as sketches like :func:`approx_set`. By
+injecting random noise into results, noisy aggregation functions make it more
+difficult to determine or confirm the exact data that was aggregated.
+
+.. note::
+
+    While many of these functions resemble
+    `differential privacy <https://en.wikipedia.org/wiki/Differential_privacy>`_
+    mechanisms, it is important to note that neither the values returned by
+    these functions nor the query results that incorporate these functions
+    are differentially private in general. See `Limitations`_ below for more
+    details. Users who wish to support a strong privacy guarantee should
```

Comment:
I'd delete " Users who wish to support a strong privacy guarantee should
    discuss with a suitable technical expert first."

Reply:
I'm going to vote to leave this sentence, as we want to be abundantly clear about both the limitations and their practical implications.